api/filter/network/http_connection_manager.proto

```diff
@@ -80,6 +80,24 @@ message HttpConnectionManager {
     // populate the tag name, and the header value is used to populate the tag value. The tag is
     // created if the specified header name is present in the request's headers.
     repeated string request_headers_for_tags = 2;
+
+    // Global target percentage of requests that will be force traced if the *x-client-trace-id*
+    // header is set. Percent is resolved to the nearest 1% (rounded down). Defaults to 1 (i.e.,
+    // 100%). This variable is a direct analog for the variable of the same name in the :ref:`HTTP
+    // Connection Manager <config_http_conn_man_runtime>`.
+    Percent client_enabled = 3;
+
+    // Global target percentage of requests that will be traced after all other checks have been
+    // applied (force tracing, sampling, etc.). Percent is resolved to the nearest 1% (rounded
+    // down). Defaults to 1 (i.e., 100%). This variable is a direct analog for the variable of the
```

Comment:
My personal lesson with infrastructure design is everything should be default to 0. Don't assume everyone wants my feature no matter how much I love it myself.

If I add a new feature to a system, I cannot touch all existing customers, so the new feature must be default off. Instead of arguing whether a new feature should be default on, we should have a hard policy, everything is default off, and don't spend time debating on it. We never know what users want to do.

Reply:
Although I agree that default 0 would be better, that's not how the current code works and we can't change it. Se la vi.